README.md

```diff
@@ -7,36 +7,39 @@ October instance.
 
 ### Dependencies
 
-- PHP 5 (version used in production)
-- [October](https://octobercms.com/)
-- Some database (MySQL is recommended)
+- [Docker](https://docker.com)
 - [Node.js](https://nodejs.org/) (6.x or newer)
   - If you cannot get a recent version from your distribution's repositories,
     try using [nvm](https://github.com/creationix/nvm).
 - [Yarn](https://yarnpkg.com/)
 
-### Installing October and the Godot theme & plugins
-
-- Download and install [October](http://octobercms.com/).
-  - Note that you don't need to install Apache or nginx, as PHP has a built-in
-    development server. You can start it with `php -S localhost:8080` which
-    will host a server in the current directory.
-- If you have trouble with the installer, try to clone the repositories instead.
-  Running `composer install` then editing `config/database.php` with your
-  database credentials should get it up and running.
-- Once installed, clone this repository into your October directory (you'll have
-  to clone into an empty directory and then move it into the October directory).
-- Reassign the `active_theme` variable in `config/cms.php` to `godotengine`.
-- You may have some errors at this point, these are most likely due to plugins.
-  If an error is thrown, it should tell you which plugin caused it. Replace
-  `authorname` and `pluginname` with the plugin you wish to run the command on.
-  - Try installing the new plugin like so:
-    `php artisan plugin:install authorname.pluginname`
-  - Or refreshing the plugin like so:
-    `php artisan plugin:refresh authorname.pluginname`
-    (warning, this will erase all the database entries!)
-  - Plugins which might need these commands run on them are `RainLab.Blog` and
-    `PaulVonZimmerman.Patreon`
+### Running the site
+
+- Clone this repo
+- Put a database dump (if you have one) into the `/docker/mariadb/init` folder
+    - Make sure that your script starts with the line `USE october;` and that the file extension is `.sql`
+- Run the `./docker/restart.sh` script (this will take a while the first time)
```

Comment:
This works fine, but I have nothing on `http://localhost:8080`. It's not clear to me if any other step should be done to access the website on my host browser.

Reply:
From reading docs on Docker Compose, it should be work out of the box.. I tried to disable my firewall, but I now get this error:
```
$ sudo ./restart.sh 
Removing network docker_godotengine-org--network
WARNING: Network docker_godotengine-org--network not found.
Creating network "docker_godotengine-org--network" with the default driver
ERROR: Failed to Setup IP tables: Unable to enable SKIP DNAT rule:  (iptables failed: iptables --wait -t nat -I DOCKER -i br-88e89f792815 -j RETURN: iptables: No chain/target/match by that name.
 (exit status 1))
```

Reply:
`systemctl restart docker` fixed it.

Reply:
I guess I'm not using the database dump properly as I get this error on localhost:
```
SQLSTATE[HY000] [2002] No such file or directory (SQL: select * from `rainlab_blog_categories` where `slug` = news order by `nest_left` asc limit 1)

/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php line 664
```